modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/incubus_draft.dm

```diff
@@ -165,36 +158,69 @@
 			new_penis.update_sprite_suffix()
 			exposed_mob.update_body()
 			to_chat(exposed_mob, span_purple("Your crotch feels warm as something suddenly sprouts between your legs."))
+			
 		// Makes the balls bigger if they're small.
 		var/obj/item/organ/external/genital/testicles/mob_testicles = exposed_mob.getorganslot(ORGAN_SLOT_TESTICLES)
 		if(mob_testicles)
-			if(mob_testicles.genital_size > 2)
-				return
-			mob_testicles.genital_size = 2
+			if(mob_testicles.genital_size < 3)
+				mob_testicles.genital_size++
+				mob_testicles.update_sprite_suffix()
+				exposed_mob.update_body()
+	
 	// Separates gender change stuff from cock growth.
-	var/obj/item/organ/external/genital/breasts/mob_breasts = exposed_mob.getorganslot(ORGAN_SLOT_BREASTS)
 	if(exposed_mob.client?.prefs?.read_preference(/datum/preference/toggle/erp/gender_change))
-		if(exposed_mob.gender == FEMALE)
-			exposed_mob.set_gender(MALE)
```

Comment:
To prevent spam from switching back and forth when overdosing on incubus draft and succubus milk together, check if 'double dosing'